test/ussd_nurse.test.js

```diff
@@ -510,6 +514,11 @@ describe("app", function() {
                         state: 'state_end_reg',
                         reply: "Thank you. Weekly NurseConnect messages will now be sent to this number."
                     })
+                    .check(function(api) {
+                        var metrics = api.metrics.stores.test_metric_store;
+                        assert.deepEqual(metrics['test.ussd_nurse.self.registration_started'].values, [1]);
```

Comment:
just change this to `test_ussd_nurse.registrations_started` - note the plural, and that I don't want to track the `self` and `other` seperately.